components/post_view/post/post.jsx

```diff
@@ -189,11 +191,24 @@ class Post extends React.PureComponent {
     }
 
     handlePostClick = (e) => {
-        const post = this.props.post;
+        const {post, isCollapsedThreadsEnabled} = this.props;
+
         if (!post) {
             return;
         }
 
+        const isSystemMessage = PostUtils.isSystemMessage(post);
+        const fromAutoResponder = PostUtils.fromAutoResponder(post);
+
+        if (
+            !e.altKey &&
```

Comment:
nit: do we need this negation here?

Reply:
@nevyangelova I think we do, alt+clicking marks the post as unread, so we don't want to open the thread on that occasion.